src/main.c

```diff
@@ -61,7 +72,17 @@ void ui_idle(void)
 	// The first argument is the starting index within menu_main, and the last
 	// argument is a preprocessor; I've never seen an app that uses either
 	// argument.
+	#if defined(TARGET_NANOS)
```

Comment:
Maybe we can move this to `ui_showMenu()` function in `menu.c` which would handle all the intricacies of different platforms. (In needs to anyway ...)

Reply:
we concluded that it's not worth it since it uses the INS_NONE constant which is needed for other functions in the file, so it would be needed to be extracted as well